gallery.css

```diff
@@ -27,6 +27,19 @@
     padding-left: 30px;
     padding-right: 30px;
 }
+.gallery-element.gallery-mobile-item {
+    display: none;
+}
+.gallery-element.gallery-desktop-item {
+    display: initial;
+}
+.gallery-element[name^=item] {
+    background: #858585; /* Fallback for browsers that do not support gradients */
+    background: -webkit-linear-gradient(left,rgba(133,133,133,1), rgba(170,170,170,1),rgba(133,133,133,1)); /*Safari 5.1-6*/
+    background: -o-linear-gradient(right,rgba(133,133,133,1), rgba(170,170,170,1),rgba(133,133,133,1)); /*Opera 11.1-12*/
+    background: -moz-linear-gradient(right,rgba(133,133,133,1), rgba(170,170,170,1),rgba(133,133,133,1)); /*Fx 3.6-15*/
+    background: linear-gradient(to right, rgba(133,133,133,1), rgba(170,170,170,1),rgba(133,133,133,1)); /*Standard*/
+}
```

Comment:
This gradient is almost invisible on larger screens. You should think about increasing the contrast, for example by changing the color stops to be closer to the edges.